resources/test/README.md

```diff
@@ -6,7 +6,7 @@ The test suite for the `testharness.js` testing framework.
 
 Install the following dependencies:
 
-- [Python 2.7](https://www.python.org/)
+- [Python 2.7.9+](https://www.python.org/)
```

Comment:
Truth of this confirmed in https://github.com/w3c/web-platform-tests/pull/8773#issuecomment-353314489